packages/cicero-server/app.js

```diff
@@ -250,6 +251,37 @@ app.post('/invoke/:template', async function(req, httpResponse, next) {
     }
 });
 
+/**
+ * Handle POST requests to /get/:template
+ * The body of the POST does not contain any argument
+ * The template is loaded using the template name
+ * The call returns the list of model files with their name and content
+ *
+ * Template
+ * ----------
+ * The template parameter is the name of a directory under CICERO_DIR that contains
+ * the template to use.
+ *
+ * Request
+ * ----------
+ * The POST body does not contain any property.
+ *
+ * Response
+ * ----------
+ * The list of models containing the names and content of .cto model file.
+ *
+ */
+app.post('/get/:template', async function (req, httpResponse, next) {
```

Comment:
The semantics of this operation are wrong, IMO. Although this corresponds to the same CLI operation `get`, in a remote API the expected behaviour would be to retrieve the template definition, and not its dependent models. Furthermore, the `/get/` resource in the path is problematic.

I suggested that we exclude this operation from the API.